Tasks/Maven/maventask.ts

```diff
@@ -21,16 +21,16 @@ if (mavenVersionSelection == 'Path') {
 tl.debug('Maven binary: ' + mvntool);
 
 var mavenPOMFile = tl.getPathInput('mavenPOMFile', true, true);
-var mavenOptions = tl.getDelimitedInput('options', ' ', false);
-var mavenGoals = tl.getDelimitedInput('goals', ' ', true);
+var mavenOptions = tl.getInput('options', false); // options can have spaces and quotes so we need to treat this as one string and not try to parse it
```

Comment:
Both will work in the main case.  Note that .arg will take a string or an array.  But yes, if it can have spaces in an arg and it's delimited by " then yes, argString is correct.